### ISSUE TITLE
[MASSEMBLY-934] Support concatenation of files

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-assembly-plugin#1142]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** opened **[MASSEMBLY-934](https://issues.apache.org/jira/browse/MASSEMBLY-934?redirect=false)** and commented

Current file entry looks like

```
<file>
      <source/>
      <outputDirectory/>
      <destName/>
      <fileMode/>
      <lineEnding/>
      <filtered/>
    </file>
```

It should be possible to define multiple sources here, to combine them into one destination file.

---
No further details from [MASSEMBLY-934](https://issues.apache.org/jira/browse/MASSEMBLY-934?redirect=false)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

You mean concatenation?

**Comment by jira-importer** [comment]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** commented

yes.

Background: the mvnw scripts are actually the mvn scripts with extras. My idea is to split them up and concat the right set of scripts, so we don't have to maintain multiple sets of the same scripts.

**Comment by jira-importer** [comment]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** commented

Fixed in [5ba9b81c27302c582e609a06865cc2e83ab2a047](https://gitbox.apache.org/repos/asf?p=maven-assembly-plugin.git;a=commit;h=5ba9b81c27302c582e609a06865cc2e83ab2a047)
